deploy/crds/sriovnetwork_v1_sriovnetworknodepolicy_crd.yaml

```diff
@@ -36,6 +36,9 @@ spec:
               - netdevice
               - vfio-pci
               type: string
+            enableSwitchDev:
+              description: Enable Switch Dev mode for Mellnaox's cards
```

Comment:
s/ Switch Dev/ Switchdev. It is not specific to mellanox ...

Reply:
It better to do enum and called it SR-IOV mode (legacy/switchdev) the default should be legacy